ingestion/strip_data.py

```diff
@@ -0,0 +1,127 @@
+    # philly_stripped = philly_stripped[["permitnumber", "permittype", "permitissuedate", "location", "contractorname"]]
+    philly_stripped = philly_stripped[["permitissuedate", "contractorname", "lat", "lng"]]
+    philly_stripped = philly_stripped.dropna(subset=["contractorname"])
+    philly_stripped["permitissuedate"] = philly_stripped["permitissuedate"].apply(lambda x: x.split(" ")[0])
+    philly_stripped.to_csv("ingestion/stripped_data/philly.csv")
+    logging.info("Saved philly.csv")
+    return philly_stripped
+
+def mesa():
+    """
+    Extracts needed columns from the Mesa data and saves it as a csv file
+    Can't filter by contractor trade
+    :return: mesa_stripped: the stripped data as a pandas dataframe
+    """
+    mesa_raw = pd.read_csv("ingestion/raw_data/mesaaz.csv")
+    mesa_stripped = mesa_raw[["issued_date", "contractor_name", "latitude", "longitude"]]
+    mesa_stripped = mesa_stripped.dropna(subset=["contractor_name"])
+    mesa_stripped["issued_date"] = mesa_stripped["issued_date"].apply(lambda x: x.split("T")[0])
+    mesa_stripped.to_csv("ingestion/stripped_data/mesaaz.csv")
+    logging.info("Saved mesaaz.csv")
+    return mesa_stripped
+
+def la():
+    """
+    Extracts needed columns from the Los Angeles data and saves it as a csv file
+    :return: la_stripped: the stripped data as a pandas dataframe
+    """
+    la_raw = pd.read_csv("ingestion/raw_data/lacity.csv")
+    la_stripped = la_raw[["issue_date", "contractors_business_name", "location_1", "permit_type"]]
+    latitude = []
+    longitude = []
+    for i in range(len(la_stripped) - 1):
+        # {'latitude': '33.99393', 'human_address': '{"address": "", "city": "", "state": "", "zip": ""}', 'needs_recoding': False, 'longitude': '-118.33429'}
+        location_object = la_stripped["location_1"][i]
+        if type(location_object) != str:
+            latitude.append("nan")
+            longitude.append("nan")
+            continue
+        location_object = location_object.split("'")
+        latitude.append(location_object[3])
+        longitude.append(location_object[13])
+    la_stripped["latitude"] = pd.Series(latitude)
+    la_stripped["longitude"] = pd.Series(longitude)
+    la_stripped = la_stripped[la_stripped.permit_type == "Electrical"]
+    la_stripped.drop(columns=["location_1", "permit_type"], inplace=True)
+    la_stripped = la_stripped.dropna(subset=["contractors_business_name"])
+    la_stripped["issue_date"] = la_stripped["issue_date"].apply(lambda x: x.split("T")[0])
+    la_stripped.to_csv("ingestion/stripped_data/lacity.csv")
+    logging.info("Saved lacity.csv")
+    return la_stripped
+
+def strip_dataframes(city_list):
+    """
+    Strips the specified dataframes and saves them as csv files
+    :param city_list: a list of the cities to strip
+    """
+    city_functions = [globals()[x] for x in city_list if x in globals() and callable(globals()[x])]
```

Comment:
Try to avoid using globals.
Also, avoid creating function calls from strings.
Either call each function on separate lines
```
mesa()
la()
austin()
```
or use a dictionary to maintain the list
```
dict = 
{
    "la": get_la_data(),
    "austin": get_austin_data()
}

for city in cities: # eg cities = ["austin", "la"] from calling function
   dict[city]()
```